omnibus/files/private-chef-ctl-commands/install.rb

```diff
@@ -4,7 +4,6 @@
 KNOWN_ADDONS = %w(
   chef-ha
   chef-sync
-  opscode-analytics
```

Comment:
Most likely. I know I've seen chatter about demos being done on a single machine even through we stopped supporting that for actually running it in production. That said, we should probably break the demos and find a solution there.